reference/tools/data-migration/dm-portal.md

```diff
@@ -0,0 +1,193 @@
+
+To address this problem, DM provides a simple web application, DM Portal. DM Portal enables users to visually configure the required replication tasks, and generates a `task.yaml` file that can be directly executed by DM.
+
+## Features
+
+### Configure the replication mode
+
+DM Portal supports three replication modes of DM:
+
+- Full replication
+- Incremental replication
+- All (full + incremental)
+
+### Configure the instance information
+
+DM Portal can configure the table routing, which supports the configuration of merging sharded schemas and tables in DM.
+
+### Configure the binlog event filter
+
+DM Portal supports configuring the binlog event filter in schemas and tables.
+
+### Generate the configuration file
+
+DM Portal can generate configuration files and download the files to the local computer. Meanwhile, it automatically creates a file in the `/tmp/` directory on the dm-portal server.
+
+## Restrictions
+
+The current DM Portal can generate visualized pages that cover most scenarios of DM configurations, but it also has some restrictions:
+
+- The SQL pattern of [binlog event filter](/reference/tools/data-migration/features/overview.md#binlog-event-filter) is not supported.
+
+- The editing feature does not support parsing the `task.yaml` file created by the user. The user can only edit the `task.yaml` file generated by the page.
+
+- The editing feature does not support modifying the instance configuration. If the user need to adjust the instance configuration, the `task.yaml` file has to be regenerated.
+
+- The upstream instance configuration on the page can only be used to obtain the upstream table schema. The related upstream instance information still needs to be configured in DM-worker.
+
+- In the generated `task.yaml` file, mydumper-path is `./bin/mydumper` by default. If you use another path, you should modify the generated `task.yaml` file manually.
+
+## Deploy
+
+### Deploy using binary
+
+You can download DM Portal at [dm-portal-latest-linux-amd64.tar.gz](https://download.pingcap.org/dm-portal-latest-linux-amd64.tar.gz). To start DM Portal, run the `./dm-portal` command.
+
+- If you run DM Portal locally, visit `127.0.0.1:8280` in your browser.
+- If you run DM Portal on a server, you need to configure a proxy on the server.
+
+### Deploy using DM Ansible
+
+To deploy DM Portal using DM Ansible, refer to [Deploy Data Migration Using DM-Ansible](/how-to/deploy/data-migration-with-ansible.md) for details.
+
+## Usage
+
+### Create rules
+
+This feature is used to create a `task.yaml` file. You need to select the replication mode, configure the upstream and downstream instances, configure table routing, and configure the binlog filter.
+
+#### Operation steps
+
+Access the dm-portal page, and click **Create New Rule**.
```

Comment:
Please check the operation order of `You need to select the replication mode, configure the upstream and downstream instances, configure table routing, and configure the binlog filter.` and **Create New Rule** to confirm which one comes first.

Reply:
Hi, I confirmed with @WangXiangUSTC that the Chinese document should be modified. I have updated this PR and will create a PR to update the Chinese doc later.

Reply:
@lilin90 PTAL https://github.com/pingcap/docs-cn/pull/2551